force-app/classes/test/AssertionsTest.cls

```diff
@@ -373,31 +373,35 @@ private class AssertionsTest {
   }
 
   @isTest
-  static void givenExpectedEqualsAssert_systemAsserter_doNotThrow() {
+  static void givenExpectedEqualsAssert_MethodSpyAsserter_doNotThrow() {
     // Arrange
-    Assertions.SystemAsserter systemAsserter = new Assertions.SystemAsserter();
+    final Assertions.Asserter sut = new Assertions.MethodSpyAsserter();
 
     // Act
-    systemAsserter.assertEquals(true, true, 'it works');
+    sut.isTrue(true, 'it works');
+    sut.isFalse(false, 'it works too');
 
     // Assert
     System.assert(true, 'This assertions should be reached');
   }
 
-  class FakeAsserter extends Assertions.SystemAsserter {
+  class FakeAsserter implements Assertions.Asserter {
     public Integer assertEqualsCallCount = 0;
```

Comment:
**suggestions:** change this assert helper and split it in 2

`assertIsTrueCallCount`
`assertIsFalseCallCount`
and then refacto where it is used